src/main/java/seedu/address/model/expenses/Money.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.model.expenses;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import seedu.address.model.expenses.exceptions.MoneyLimitExceededException;
+
+/**
+ * Class to represent monetary values.
+ * Ensures that prices keyed in for Expense objects are valid.
+ * Range of dollars limited to +/- Integer.MAX_VALUE.
+ * Range does not include Integer.MIN_VALUE to prevent errors when changing signs.
+ */
+public class Money {
+
+    private int dollars;
+    private int cents;
+    //v2.0: implement exchange rates
+
+    public Money(int dollars, int cents) {
+        if (cents >= 100 || cents < 0 || dollars == Integer.MIN_VALUE) {
+            throw new IllegalArgumentException();
+        }
+        this.dollars = dollars;
+        this.cents = cents;
+    }
+
+    /**
+     * Converts a string with valid money format into a Money object.
+     * @param toConvert The string to convert to a Money object.
+     */
+    public Money(String toConvert) {
+        if (!isValidMoneyFormat(toConvert)) {
+            throw new IllegalArgumentException();
+        }
+        String[] parts = toConvert.split("\\.");
+        dollars = Integer.parseInt(parts[0]);
+        cents = Integer.parseInt(parts[1]);
+    }
+
+    /**
+     * Checks that a string is a valid representation of money.
```

Comment:
Please write documentation for what a valid Money object looks like as a string. For example, see `RoomNumber`.